iquip/apps/explorer.py

```diff
@@ -113,19 +156,20 @@ def openExperiment(self):
         experimentFileItem = self.explorerFrame.fileTree.currentItem()
         if experimentFileItem.childCount():
```

Comment:
Hmm.. this becomes now a problem, since during a directory is loaded, the widget item will have no child.

Reply:
Umm... it is a critical issue.

Then, how about not checking at all?
Actually, even if we completely check that the selected element is a file, we should re-check whether the file is an ARTIQ experiment file later.
This will be processed through an ARTIQ command line (probably, customed), so why don't we postpone it to this time?

Reply:
If you want to check it in this branch, we can do this through comparing argumetns of `self.thread` to `experimentFileItem`.

Reply:
As you said, later we have to check more things. I think it is fine not to check at all for now!